multigeiger/multigeiger.ino

```diff
@@ -223,23 +222,21 @@ volatile unsigned long isr_count_timestamp_2send= micros();
          float         bme_pressure           = 0.0;
          float         GMC_factor_uSvph       = 0.0;
          char          sw[4]                  = {0,0,0,0};
-
+         portMUX_TYPE  mux = portMUX_INITIALIZER_UNLOCKED;
 
 int Serial_Print_Mode = SERIAL_DEBUG;
 
 //====================================================================================================================================
 // ISRs
-void isr_GMC_capacitor_full() {
+void IRAM_ATTR isr_GMC_capacitor_full() {
   GMC_cap_full = 1;
```

Comment:
please add a comment here about why acquiring/releasing the portmux is not needed here.

Reply:
see my comment in #74.

looks like we need the portENTER_CRITICAL_ISR(&mux) + ...EXIT... calls here.

also i suggest that we use 2 different MUXes:
- one for the GM pulse counter resource
- one for the HV capacitor full resource.